mysql-test/suite/innodb/r/bulk_load.result

```diff
@@ -48,3 +49,15 @@ CHECK TABLE t2 EXTENDED;
 Table	Op	Msg_type	Msg_text
 test.t2	check	status	OK
 DROP TABLE t2, t1;
+#
+#  MDEV-37192  Crash recovery after bulk load data
+#                reports corruption
```

Comment:
I find the bug title hard to parse. I would suggest the following, to follow the conventional English grammar ([subject, predicate, object, place, time](https://readle-app.com/en/grammar/english/a1/basic-word-order-in-english-sentences/)) regarding [word order](https://www.english-4u.de/en/grammar/word-order.html):
>Crash recovery reports corruption after bulk load